wandb/sdk/lib/redirect.py

```diff
@@ -629,6 +616,47 @@ def uninstall(self):
         super().uninstall()
 
 
+class StreamRawWrapper(RedirectBase):
+    """
+    Patches the write method of current sys.stdout/sys.stderr
+
+    Captures data in a raw form rather than using the emulator
+    """
+
+    def __init__(self, src, cbs=()):
+        super().__init__(src=src, cbs=cbs)
+        self._installed = False
+
+    def install(self):
+        super().install()
+        if self._installed:
+            return
+        stream = self.src_wrapped_stream
+        old_write = stream.write
+        self._prev_callback_timestamp = time.time()
+        self._old_write = old_write
+
+        def write(data):
+            self._old_write(data)
+            for cb in self.cbs:
+                try:
+                    cb(data)
+                except Exception as e:
+                    # TODO: remove me
```

Comment:
what is this todo?

Reply:
my attempt to better debug what was going on with these bare try/accepts, this is from the existing code..
I will remove the write and we will swallow silently any exceptions like the old code did... for now.  ill update the todo to say we should do something here.

Reply:
done